src/App.js

```diff
@@ -19,17 +19,17 @@ class App extends Component {
     this.props.getUserInfo();
   }
 
-  componentWillReceiveProps(newProps) {
-    if (newProps.isAuthed !== this.props.isAuthed && !newProps.isAuthed) {
-      window.location.href = `${rootPath}/auth/devmtn`;
-    }
-  }
+  // componentWillReceiveProps(newProps) {
+  //   if (newProps.isAuthed !== this.props.isAuthed && !newProps.isAuthed) {
+  //     window.location.href = `${rootPath}/auth/devmtn`;
+  //   }
+  // }
 
```

Comment:
Undo these changes, with new modifications you will now only have to login when server is restarted, so this is no longer necessary.